### PR TITLE
Clarify list applies to success actions

### PR DESCRIPTION
**Pull request by jcrben**
Closes https://github.com/jmeas/redux-resource/issues/255

### CODE REVIEW

File: docs/guides/lists.md [hunk 1]
```diff
@@ -12,23 +12,24 @@ more in-depth in this guide, but first, let's cover how to use lists.
...
 import { actionTypes } fom 'redux-resource';
 import store from './store';
 
 store.dispatch({
-  type: actionTypes.READ_RESOURCES_PENDING,
+  type: actionTypes.READ_RESOURCES_SUCCEEDED,
   resourceName: 'books',
   list: 'mostPopular',
-  // Remember, we always want to track the request status
-  request: 'getMostPopular'
```
Comment (jmeas) [inline docs/guides/lists.md]: We should keep the request on the action, I think. We may still need it to show a success message somewhere!

**Comment by jmeas** [comment]
Lookin’ good. I left one small comment, then we can get this update merged and deployed.

**Comment by jmeas** [comment]
After the above change is made, I think this is good enough to merge, but I had one more thought related to this topic. It may be mentioning that it's _alright_ to send the `list` along for non-success actions. Nothing will break if you do that.

The reason that that's worth mentioning is that it's usually economical to do that when using a library that helps construct the action creators, such as [redux-resource-xhr](https://redux-resource.js.org/docs/extras/redux-resource-xhr.html). That just sends along whatever attributes you pass in, including the `list`, to all of the actions associated with that request. This makes the library faster and easier to understand than whitelisting or blacklisting fields, even if it's a little less exact.

It may also be a good practice just so that it provides more information for [plugins](https://redux-resource.js.org/docs/guides/plugins.html) that might do some fancy stuff with [list metadata](https://github.com/jmeas/redux-resource/issues/249).

---

Feel free to throw in a note about any of this into this PR, @jcrben , or I can do a separate PR after this one lands. Nbd either way. Thanks again for the help :v:

**Comment by jcrben** [comment]
OK, I'll see about massaging the PR. 

Once the action-creators thing is refined that'll  probably happen naturally. I'm trying redux-loop which works with promises so redux-resources-xhr wasn't a good fit - using axios for requests.

**Comment by jmeas** [comment]
> OK, I'll see about massaging the PR.

:+1:

> Once the action-creators thing is refined that'll probably happen naturally. I'm trying redux-loop which works with promises so redux-resources-xhr wasn't a good fit - using axios for requests.

Oh cool. I hadn’t heard of redux-loop. Let us know if there are any changes that can be made to this library to make it integrate with loop more easily.

Also, if you end up writing an abstraction similar to redux-resource-xhr for axios, and want it added to this lib, we’d [be happy to merge that in](https://github.com/jmeas/redux-resource/issues/31) :v:

**Comment by jcrben** [comment]
Made some edits, feel free to edit it up at your leisure. Figured I'd reuse the existing callout note.

redux-loop seems to be working OK so far altho I'm just noodling around. I was thinking I would just use https://github.com/jmeas/redux-resource/issues/186 to start for abstracting over

on an unrelated topic, what node and npm version are you using for the build? when I tried `npm run bootstrap` running node@7.9.0 and npm@5.5.1 my computer froze up for a few minutes until I managed to kill the process

I like to put a `.node-version` in my projects (nodenv respects it, nvm will soon) - may look at running my npm installs thru a docker executable container to lock things down further since npm has been flakey lately

**Comment by jmeas** [comment]
Whoops – I missed this update somehow. Sorry about that. I'll take a look at this soon, although I may be away from the computer for the next few days :v:

**Comment by jmeas** [comment]
Sorry for the delay, @jcrben ! This looks great. Merging and deploying now!

**Pull request merged**